plugins/navigation-files-transformer/gatsby-node.js

```diff
@@ -49,11 +45,25 @@ const onCreateNode = async (api, options) => {
       children: [],
       parent: node.id,
       internal: {
+        content: "",
         contentDigest: createContentDigest(parsedContent),
-        type: "parliamentNavigation",
+        type: "ParliamentNavigation",
       },
     }
     createNode(navigationNode)
+    createParentChildLink({ parent: node, child: navigationNode })
+  }
+}
+
+exports.setFieldsOnGraphQLNodeType = args => {
```

Comment:
JSDocs would be helpful here

Reply:
It's from https://www.gatsbyjs.org/docs/node-apis/#setFieldsOnGraphQLNodeType

Reply:
Added JsDocs